.github/actions/component-diff/action.yaml

```diff
@@ -0,0 +1,111 @@
+name: component-diff
+description: |
+  An opinionated action to determine the component-diff between the current version (read from
```

Comment:
I think it would fit in better if this action would need the (resolved / merged) component-descriptor as an input, and if the convenience-workflow wrapping it would deal w/ collecting the component-descriptor from `prepare.yaml` workflow. This way, the action could be instrumented differently (i.e. w/o implemanting against how `prepare.yaml` exposes base-component-descriptor), while still being convenient (for users of wrapping workflow).